doc/release-notes/upgrading.en.rst

```diff
@@ -64,14 +64,19 @@ Configuration changes
 ---------------------
 The following incompatible changes to the configurations have been made in this version of ATS.
 
-The records.yaml entry proxy.config.http.down_server.abort_threshold has been removed.
-
-The records.yaml entry proxy.config.http.connect_attempts_max_retries_dead_server has been renamed to proxy.config.http.connect_attempts_max_retries_down_server.
-
-The records.yaml entry proxy.config.http.connect.dead.policy has been renamed to proxy.config.http.connect.down.policy.
-
-The records.yaml entry proxy.config.http.parent_proxy.connect_attempts_timeout and proxy.config.http.post_connect_attempts_timeout
-have been removed. Instead use proxy.config.http.connect_attempts_timeout to control all connection to origin timeouts.
+- The records.yaml entry ``proxy.config.http.down_server.abort_threshold`` has been removed.
+- The records.yaml down server policy was sometimes called "dead" server and sometimes "down" server. The configuration
+  names have been consolidated around "down" server.
+
+  - The records.yaml entry ``proxy.config.http.connect_attempts_max_retries_dead_server`` has been renamed to ``proxy.config.http.connect_attempts_max_retries_down_server``.
```

Comment:
Is there an extra whitespace at the beginning?

Reply:
That makes them sub-bullets under the down/dead server topic. Here's the rendered output:

https://ci.trafficserver.apache.org/job/Github_Builds/job/docs/5650/artifact/output/11679/docbuild/html/release-notes/upgrading.en.html#configuration-changes

Reply:
Removed the sub-bullets to make this consistent with the other configuration changes.